static/directives/quay-message-bar.html

```diff
@@ -1,10 +1,17 @@
-<div class="announcement inline quay-message-bar-element" ng-show="messages.length || inReadOnlyMode">
+<div class="announcement inline quay-message-bar-element" ng-show="messages.length || inReadOnlyMode || inAccountRecoveryMode">
   <div class="quay-service-status-description info" ng-if="inReadOnlyMode">
     <div style="display: inline-block">
       <span class="registry-name"></span>&nbsp;is currently in read-only mode. Pulls and other read-only operations
       will succeed but all other operations are currently suspended.
     </div>
   </div>
+  <div class="quay-service-status-description info" ng-if="inAccountRecoveryMode">
```

Comment:
Can we increase the severity from `info` to `warning`

Reply:
Done